Tests/Editor/GameLiftPlugin/AnywherePageTests.cs

```diff
@@ -0,0 +1,176 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Amazon.GameLift.Model;
+using AmazonGameLift.Editor;
+using AmazonGameLiftPlugin.Core.ApiGatewayManagement;
+using AmazonGameLiftPlugin.Core.SettingsManagement.Models;
+using Moq;
+using NUnit.Framework;
+
+namespace Editor.Window.UnitTests
+{
+    [TestFixture]
+    public class AnywherePageTests
+    {
+
+        private Mock<GameLiftPlugin> _gameLiftConfigMock;
+        private Mock<IAmazonGameLiftClientWrapper> _gameLiftWrapperMock;
+        private Mock<IAwsCredentialsFactory> _awsCredentialsFactory;
+        private Mock<CoreApi> _coreApi;
+        private Mock<IAmazonGameLiftClientFactory> _amazonGameLiftClientFactory;
+        
+        [SetUp]
+        public void Setup()
+        {
+            _gameLiftConfigMock = new Mock<GameLiftPlugin>();
+            _gameLiftWrapperMock = new Mock<IAmazonGameLiftClientWrapper>();
+            _awsCredentialsFactory = new Mock<IAwsCredentialsFactory>(); 
+            _coreApi  = new Mock<CoreApi>();
+            _amazonGameLiftClientFactory = new Mock<IAmazonGameLiftClientFactory>();
+        }
+        
+        private readonly TextProvider _textProvider = TextProviderFactory.Create();
+
+        private AwsCredentials GetAwsCredentialsWithStubComponents(CoreApi coreApi)
```

Comment:
Anything that uses an AwsCredentials object will likely need a similar method, consider splitting this into a provider utility (ex: under `Tests/Utils/Providers/AwsCredentialsProvider.cs`)